_typos.toml

```diff
@@ -1,5 +1,6 @@
 [default.extend-words]
 ba = "ba"
+availabilty = "availabilty"
```

Comment:
This is a typo from cairo

Reply:
How about we make them a fix typo PR? 😏 
I think it would be nice to have it fixed